testsuite/features/profiles/Docker/add_packages.sh

```diff
@@ -12,7 +12,7 @@ cp /root/avahi-daemon.conf /etc/avahi/avahi-daemon.conf
 
 # re-enable normal repo and remove helper repo
 zypper mr --enable Fake-RPM-SUSE-Channel || :
-zypper rr sles15sp4
+zypper rr sles15sp7
```

Comment:
What about `features/profiles/Docker/sles15sp4.repo`?
I don't see it replace it on this PR.

Same for:
-  `testsuite/features/profiles/Docker/Dockerfile`
- all the ones in the nested folder `features/profiles/Docker/serverhost` and `features/profiles/Docker/authprofile`
- and all the sibbling folders like `features/profiles/github_runner` or `features/profiles/internal_nue`

Reply:
I will check with temporary. Maybe we should just point to temporary